docs/outlook/debug-ui-less.md

```diff
@@ -1,16 +1,19 @@
 ---
-title: Debug your UI-less Outlook add-in
-description: Learn how to debug your UI-less Outlook add-in.
+title: Debug function commands in Outlook add-ins
+description: Learn how to debug function commands in Outlook add-ins.
 ms.topic: article
-ms.date: 05/19/2022
+ms.date: 07/11/2022
 ms.localizationpriority: medium
 ---
 
-# Debug your UI-less Outlook add-in
+# Debug function commands in Outlook add-ins
 
-This article describes how to use the Office Add-in Debugger Extension in Visual Studio Code to debug [UI-less Outlook add-ins](add-in-commands-for-outlook.md#executing-a-javascript-function). UI-less add-in actions are initiated through an add-in command button in the ribbon. For more information about add-in commands, see [Add-in commands for Outlook](add-in-commands-for-outlook.md).
+> [!NOTE]
+> The technique in this article can be used only on a Windows development computer. If you're developing on a Mac, see [Debug function commands](../testing/debug-function-command.md).
```

Comment:
```suggestion
> The technique in this article can only be used on a Windows development computer. If you're developing on a Mac, see [Debug function commands](../testing/debug-function-command.md).
```

Reply:
I think the original is correct. What is limited is WHERE it can be used (Windows but not Mac), not WHAT you can do with it (use it but not admire it or eat it or ...).